src/main/java/gregtech/common/pipelike/itempipe/net/ItemNetHandler.java

```diff
@@ -144,39 +153,65 @@ public ItemStack insertRoundRobin(ItemStack stack, boolean simulate, boolean glo
      * Inserts items equally to all handlers
      * if it couldn't insert all items, the handler will be removed
      *
-     * @param copy     to insert to
-     * @param stack    to insert
-     * @param simulate simulate
+     * @param copy     where to insert to
+     * @param stack    what to insert
+     * @param simulate whether to actually insert or not
      * @return remainder
      */
-    private ItemStack insertToHandlers(List<ItemRoutePath> copy, ItemStack stack, boolean simulate) {
+    private ItemStack insertEquallyToHandlers(List<ItemRoutePath> copy, ItemStack stack, boolean simulate) {
         Iterator<ItemRoutePath> routePathIterator = copy.listIterator();
         int inserted = 0;
         int count = stack.getCount();
-        int c = count / copy.size();
-        int m = c == 0 ? count % copy.size() : 0;
```

Comment:
does anyone know what `m` is meant to do here? I wasn't sure when reading the code/context of the code around it

Reply:
It's how many items couldn't be split evenly by just dividing them between the inventories. A remainder to be distributed one by one in the while loop. 
At least i think it is. It's been a while since the last time i looked at this.